Explorer/Assets/DCL/AvatarRendering/Wearables/Components/WearableThumbnailComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace DCL.AvatarRendering.Wearables.Components
+{
+    public struct WearableThumbnailComponent
```

Comment:
I have a feeling this component is redundant:
- it is created from the DTO systems (from the same domain)
- they could call a common method to create `GetTexturePromise`, you don't really need a separate query (see `StartWearableThumbnailDownload`)
- You can probably add that promise to the same entity (you don't need a separate entity to hold the promise structure itself), it should not break the flow of queries